setup.py

```diff
@@ -54,8 +54,8 @@ def read(fname):
 
       # Declare minimal set for installation
       install_requires=['boto3>=1.9.38', 'numpy>=1.9.0', 'protobuf>=3.1', 'scipy>=0.19.0',
-                        'urllib3 >=1.21, <1.23',
-                        'PyYAML>=3.2', 'protobuf3-to-dict>=0.1.5', 'docker-compose>=1.21.0'],
+                        'urllib3 >=1.21', 'PyYAML>=3.2', 'protobuf3-to-dict>=0.1.5',
+                        'docker-compose>=1.23.0'],
```

Comment:
not https://github.com/docker/compose/releases/tag/1.23.1 ?

Reply:
1.23 is enough :)